### PR TITLE
Use of short syntax in case of compilation of Volt's templates

### PR DESCRIPTION
**Pull request by Studentsov**
Use of syntax of arrays and "echo" in PHP 5.4 style

### CODE REVIEW

File: phalcon/mvc/view/engine/volt/compiler.zep [hunk 1]
```diff
@@ -1779,10 +1779,10 @@ class Compiler implements InjectionAwareInterface
 		 * Echo statement
 		 */
 		if this->_autoescape {
-			return "<?php echo $this->escaper->escapeHtml(" . exprCode . "); ?>";
+			return "<?= $this->escaper->escapeHtml(" . exprCode . ") ?>";
 		}
 
-		return "<?php echo " . exprCode . "; ?>";
+		return "<?= " . exprCode . " ?>";
```
Comment (sergeyklay) [inline phalcon/mvc/view/engine/volt/compiler.zep]: What exactly Phalcon should do here if short syntax is disabled?

**Comment by sergeyklay** [comment]
As you can see, tests fails.

**Comment by Studentsov** [comment]
http://php.net/manual/en/migration54.new-features.php
`<?= is now always available, regardless of the short_open_tag php.ini option. `
Conditions of tests need to be changed, I didn't consider it.

**Comment by sergeyklay** [comment]
Really?

```sh
$ php -i | grep short
short_open_tag => Off => Off
```

**Comment by sergeyklay** [comment]
Also user/hoster/provider/someone else can disble it

**Comment by Studentsov** [comment]
Use of this syntax doesn't depend now on settings in any way. Above I gave the reference to official documentation.

**Comment by sergeyklay** [comment]
Anyway I don't understand what issue aims to solve your PR.

**Comment by Studentsov** [comment]
Only abbreviation of quantity of a code on an output from the compiler and coercion to PHP 5.4 style

File: phalcon/mvc/view/engine/volt/compiler.zep (hunk 1 above)
Comment (sergeyklay) [inline phalcon/mvc/view/engine/volt/compiler.zep]: what issue will be solved by removing the semicolon?

Comment (Studentsov) [inline phalcon/mvc/view/engine/volt/compiler.zep]: Existence or absence of a semicolon doesn't influence behavior in this case. Moreover, in official documentation such option of syntax is given in such look: http://php.net/manual/en/function.echo.php

**Pull request closed without merging**